pact/coin-contract/v2/install-coin-contract-v2.yaml

```diff
@@ -0,0 +1,3 @@
+codeFile: coin-install.pact
+nonce: coin-contract-v2-temp
```

Comment:
this changes the nonce for v2. Why?

Reply:
We already made this change for the new 10 chains' genesis blocks, on mainnet and devnet. We just did it in `Ea.gen20ChainsPayloads`, by munging the YAML programmatically. This change just makes things more uniform, and avoids the YAML munging.